docs/sources/mimir/manage/tools/mimirtool.md

```diff
@@ -571,16 +571,27 @@ Only one of the namespace selection flags can be specified.
 Grafana Mimir exposes a [remote read API] which allows the system to access the stored series.
 The `remote-read` subcommand `mimirtool` enables you to interact with its API, and to determine which series are stored.
 
+The remote-read commands support multiple `--selector` flags to query multiple series selectors in a single request, leveraging the remote read protocol's native batching capabilities for improved performance.
```

Comment:
Should we explain that there will still be just one combined set of output? (eg. for `stats`, you'll get a single set of statistics for all selectors, not a set of statistics per selector)